### DISCUSSION TITLE
Kinematic Viscosity for Weiler 2018

### DISCUSSION THREAD

**Discussion by IvanPombo** [InteractiveComputerGraphics/SPlisHSPlasH#240] [Q&A]
First of all, thanks for all the effort to established this wonderful and complex software.

From your documentation and the code itself, it seems that the viscosity parameter for the viscosity solver of Weiler 2018 is the kinematic viscosity parameter.

However, when I insert the correct physical parameters for honey, density=1400 Kg/m^3 and kinematic viscosity=9.9e-3 m^2/s(taken from https://www.engineersedge.com/fluid_flow/kinematic-viscosity-table.htm), I don't obtain a viscous flow as expected. 

In fact, I obtain almost a flow identical to water. When I insert the dynamic viscosity=14 Pa.s the visual results are way more close to what is expected.

Can you clarify this parameter for me? Is the parameter not physical at all?

**Comment by janbender** [comment]
The parameter is physical meaningful. However, the viscosity parameter is scale dependent. We had a similar discussion before: 

https://github.com/InteractiveComputerGraphics/SPlisHSPlasH/discussions/170

In this case the problem was that in our example coiling scene the scale is not a typical honey scale. The fluid block is 12m high. So can you check the dimension of your scene?
